include/zephyr/drivers/lora.h

```diff
@@ -66,6 +66,21 @@ struct lora_modem_config {
 
 	/* Set to true for transmission, false for receiving */
 	bool tx;
+
+	/* Invert the IQ. Normally this should be set to false. In advanced
```

Comment:
It might be worthwhile being explicit with In-Phase and Quadrature naming.
`Invert the In-Phase and Quadrature (IQ) signals.` That would give people a better starting point to googling additional info